pkger/service_models.go

```diff
@@ -19,7 +19,8 @@ type stateCoordinator struct {
 	mTelegrafs  map[string]*stateTelegraf
 	mVariables  map[string]*stateVariable
 
-	labelMappings []stateLabelMapping
+	labelMappings         []stateLabelMapping
+	labelMappingsToRemove []stateLabelMappingForRemoval
```

Comment:
separating these two, trying to juggle them with the same type felt icky. Side effect of this decision, is we can now simplify the service implementation dramatically.